bin/pygrb/pycbc_pygrb_minifollowups

```diff
@@ -0,0 +1,205 @@
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+"""
+Temporary script to produce qscans of loudest triggers/missed injections
+"""
+
+# =============================================================================
+# Preamble
+# =============================================================================
+import os
+import argparse
+import logging
+import h5py
+import numpy as np
+from pycbc import init_logging
+import pycbc.workflow.minifollowups as mini
+import pycbc.version
+import pycbc.workflow as wf
+import pycbc.events
+from pycbc.results import layout
+from pycbc.results.pygrb_postprocessing_utils import get_grb_time
+from pycbc.results.pygrb_postprocessing_utils import extract_ifos
+from pycbc.workflow.plotting import PlotExecutable
+
+__author__ = "Francesco Pannarale <francesco.pannarale@ligo.org>"
+__version__ = pycbc.version.git_verbose_msg
+__date__ = pycbc.version.date
+__program__ = "pycbc_pygrb_minifollowupss"
+
+
+# =============================================================================
+# Functions
+# =============================================================================
+def add_wiki_row(outfile, cols):
+    """
+    Adds a wiki-formatted row to an output file from a list or a numpy array.
+    """
+    with open(outfile, 'a') as f:
+        f.write('||%s||\n' % '||'.join(map(str, cols)))
+
+
+def make_timeseries_plot(workflow, snr_type, central_time, shift_time, out_dir,
+                         ifo=None, tags=None):
+    """Adds a node for a timeseries of PyGRB results to the workflow"""
+
+    tags = [] if tags is None else tags
+
+    # Initialize job node with its tags
+    grb_name = workflow.cp.get('workflow', 'trigger-name')
+    extra_tags = ['GRB'+grb_name]
+    extra_tags += [snr_type]
+    if ifo is not None:
+        extra_tags += [ifo]
+    node = PlotExecutable(workflow.cp, 'pygrb_plot_snr_timeseries',
+                          ifos=workflow.ifos, out_dir=out_dir,
+                          tags=tags+extra_tags).create_node()
+    trig_file_path = workflow.cp.get('workflow', 'trig-file')
+    node.add_opt('--trig-file', trig_file_path)
+    # Veto directory and segments directory
+    node.add_opt('--veto-directory', workflow.cp.get('workflow', 'veto-directory'))
+    node.add_opt('--segment-dir', workflow.cp.get('workflow', 'segment-dir'))
```

Comment:
This is probably a FIXME for the future, but it's not best practice to send files through like this. It assumes a shared file system (which is normally true in practice, but doesn't have to be). Better to have this as explicit input files. The workflow generator itself always runs on a head node, so can assume this safely, but this code will run on a compute node .... As will work in most instances as is, this is probably low down the priority tree, but adding a FIXME into the code itself here might be a good idea.